projects/packages/forms/src/contact-form/js/date-picker.js

```diff
@@ -0,0 +1,6 @@
+import Datepicker from 'vanillajs-datepicker/Datepicker';
+
+import 'vanillajs-datepicker/css/datepicker.css';
+
+const elem = document.querySelector( 'input.jp-contact-form-date' );
```

Comment:
Assuming you should create a nodeList here to loop through since there could be many date inputs in the form?